clang/lib/StaticAnalyzer/Core/MemRegion.cpp

```diff
@@ -1022,6 +1022,23 @@ getStackOrCaptureRegionForDeclContext(const LocationContext *LC,
   return (const StackFrameContext *)nullptr;
 }
 
+static bool isStdStreamVar(const VarDecl *D) {
```

Comment:
BTW wouldn't this code accept a `mynamespace::stdin`? I could not see any places that would expect a TU decl context (aka. global namespace).